ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -313,52 +312,51 @@ class GroundwaterFlowProcess : public Process
     {
         DBUG("Postprocessing GroundwaterFlowProcess.");
 
-#ifdef USE_PETSC // Note: this is only a test
-
-        std::vector<PetscScalar>  u(_x->size());
-        _x->getGlobalVector(&u[0]);
-
-        int rank;
-        MPI_Comm_rank(PETSC_COMM_WORLD, &rank);
-        if(rank == 0)
-        {
-            std::string output_file_name = file_name;
-            boost::erase_last(output_file_name, ".vtu");
-            output_file_name.append(".dat");
-            std::ofstream os(output_file_name);
-            os << "SCALARS HEAD double 1"<<endl;
-            os << "LOOKUP_TABLE default"<< endl;
-            for (std::size_t i = 0; i < u.size(); ++i)
-                 os << u[i] << "\n";
-            os.close();
-        }
-#else
-        std::string const property_name = "Result";
+        std::string const property_name = "Head [m]";
```

Comment:
One can not assume that the head is computed in meter. Could be feet or dimensionless.
Please move this to another PR.